python-sdk/src/astro/sql/operators/base_decorator.py

```diff
@@ -86,6 +86,14 @@ def execute(self, context: Context) -> None:
                 raise ValueError("You need to provide a table or a connection id")
         self.database_impl = create_database(self.conn_id, first_table)
 
+        # currently, cross database operation is not supported
+        if (
+            (first_table and self.output_table)
```

Comment:
Same question as above.

Reply:
These are not required param always in the case of BaseSQLDecoratedOperator